playwright-e2e/playwright.config.ts

```diff
@@ -28,8 +28,7 @@ const testConfig: PlaywrightTestConfig = {
     toMatchSnapshot: {
       // Account for minor difference in text rendering and resolution between headless and headed mode
       threshold: 0.3,
-      maxDiffPixelRatio: 0.2,
-      maxDiffPixels: 200
```

Comment:
Specify all three rules meant that all rules must be met in order for the screenshot comparison to pass. `maxDiffPixels: 200` is not needed because it could contradict with other two rules.